boards/riscv32/arty_litex_vexriscv/arty_litex_vexriscv.yaml

```diff
@@ -0,0 +1,10 @@
+identifier: arty_litex_vexriscv
+name: LiteX SoC with VexRiscV softcore CPU
+type: mcu
+arch: riscv32
+toolchain:
+  - zephyr
+testing:
```

Comment:
Would be good to add the amount of memory / flash here.

Reply:
There is no flash on the board, but we will add the ram size.